CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added an environment variable to control whether or not sensitive data will be masked in the console output.<br/>
```

Comment:
It's just so that the GH version looks a little nicer instead of having to add bullet points to the description or make the description of the enhancement a code block